x-pack/plugins/security_solution/public/overview/components/detection_response/rule_alerts_table/use_rule_alerts_items.test.ts

```diff
@@ -102,7 +98,6 @@ describe('useRuleAlertsItems', () => {
   it('should return new updatedAt', () => {
     const newDateNow = new Date('2022-04-08T14:00:00.000Z').valueOf();
```

Comment:
Can we chat about this, I didnt quite follow how it forced a rerender with the new value, or if its using the past tests to have the old value before switching the value in this test.

Reply:
Here is an example of `mockReturnValueOnce` https://jestjs.io/docs/mock-function-api#mockfnmockreturnvalueoncevalue

On the first call of Date.now it returns dateNow setup in Line 101, after that Date.now all always return newDateNow setup in Line 100.